Makefile

```diff
@@ -39,28 +41,37 @@ static-doc:
 # but also ensures targets like benchmarks remain buildable.
 # NOTE: excludes must match test-workspace nextest params,
 #       otherwise some rebuilding will occur due to resolver = "2"
-build-workspace:
+build-workspace-wasmer_sys:
+	$(F) cargo build \
+		--workspace \
+		--locked \
+		--all-targets \
+		--no-default-features \
+		--features $(DEFAULT_FEATURES),wasmer_sys
+
+build-workspace-wasmer_wamr:
 	$(F) cargo build \
 		--workspace \
 		--locked \
-		--exclude hdk_derive \
-		--features $(A) --all-targets
+		--all-targets \
+		--no-default-features \
+		--features $(DEFAULT_FEATURES),wasmer_wamr
 
 # execute tests on all creates
-test-workspace:
+test-workspace-wasmer_sys:
 	cargo install cargo-nextest
 	$(F) RUST_BACKTRACE=1 cargo nextest run \
 		--workspace \
-		--exclude hdk_derive \
 		--locked \
-		--features $(A)
-	# hdk_derive cannot currently be tested via nextest
```

Comment:
Is this working now?

Reply:
I guess the CI will answer this question so this is partly left here to remind me to look :)

Reply:
Yes the upstream issue was resolved: https://github.com/nextest-rs/nextest/issues/267#issuecomment-2128030529

Reply:
woo!